public/index.html

```diff
@@ -2,10 +2,246 @@
 <html>
 <head>
   <meta charset="utf-8">
-  <title>06-tshirts</title>
+  <title>Tshirt Layout</title>
   <link rel="stylesheet" href="/app.css">
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
 </head>
 <body>
+  <div class="outer-container">
+    <div class="container">
```

Comment:
It doesn't look you have styled `container` or `outer-container`.

Look at the lesson on simple grids and layouts to see how these could be styled to match the double container pattern: https://online.theironyard.com/paths/579/units/3686/lessons/15005